### DISCUSSION TITLE
Have "Max battery charge" survive reboot

### DISCUSSION THREAD

**Discussion by Cheule** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#77] [Ideas]
It seems the "Max Battery charge" I set in Al Dente v1.2 does not survive a reboot.  Upon reboot it is at maximum value again (103% with MacOS Battery Scale checked).  I'm glad the app has an option to launch at restart, and it would be great if the app read and saved to a configuration file.

In case this is already a feature, and it's simply not working for me, here is my configuration:
MacOS Big Sur 11.1
MacBook Pro (15-inch, 2019)

Thank you.

**Comment by MatthiasKerbl** [comment]
Hey Cheule, do you still have this issue?

**Reply by Cheule** [reply]
I don't, but i'm not sure what changed!  It's surviving reboot now.

**Reply by openui54u** [reply]
It is not surviving reboot with me. Would be great if this is fixed. Great software by the way. Thanks!!!

**Reply by openui54u** [reply]
Info : MacBook 15 inch 2018, 2.6Ghz 6-Core Intel Core i7, MacOS Catalina ( 10.15.7 ). Booting on internal or external SSD.

**Reply by Cheule** [reply]
I take it back, the SOC % did not survive a recent reboot.  I didn't even notice it had gone back up to 103%, which is annoying lol because now I have to drain the battery again :)
